ckanext/ytp/comments/email_notifications.py

```diff
@@ -209,7 +187,9 @@ def notify_admins_and_comment_notification_recipients(owner_org, user, template,
             users,
             template,
             {
-                'url': get_content_item_link(content_type, content_item_id, comment_id)
+                'url': get_content_item_link(content_type, content_item_id, comment_id),
+                'content_item_title': content_item_title,
```

Comment:
why not remove html markup from item title, better safe their sorry.

Reply:
@duttonw removed html markup from item title to play it safe